flex-os.conf: Fix ARCHIVE_RELEASE_VERSION

pragow0k opened the pull request.

pragow0k committed:
flex-os.conf: Fix ARCHIVE_RELEASE_VERSION

ARCHIVE_RELEASE_VERSION = "${DISTRO_VERSION}.${BSP_VERSION}.${PATCH_VERSION}"

The above line was adding extra .0.0 to
archive files since distro version is
synced with Release_version which was in
*.*.* formate, so above assigned value was
unnecessary and so being removed.

Jira_ID: SB-23809

Signed-off-by: Prajwal <prajwal.gowda-hm@siemens.com>

hashraf91 commented on meta-flex-distro/conf/distro/flex-os.conf:
```diff
@@ -17,7 +17,7 @@ QUARTER_VERSION = "04"
 RELEASE_VERSION = "${YEAR_VERSION}.${QUARTER_VERSION}.${UPDATE_VERSION}"
 DISTRO_VERSION = "${RELEASE_VERSION}"
-ARCHIVE_RELEASE_VERSION = "${DISTRO_VERSION}.${BSP_VERSION}.${PATCH_VERSION}"
+ARCHIVE_RELEASE_VERSION = "${DISTRO_VERSION}"
```
I think we can also remove these variables now:
BSP_VERSION
PATCH_VERSION

pragow0k replied:
removed it from the script

Commit:
Remove BSP_VERSION and PATCH_VERSION variable from script

use of variable in this script is droped in commit 852a3501c7ef3b0e980a027a4a932745f56d2f41

Signed-off-by: Prajwal <prajwal.gowda-hm.ext@siemens.com>
